packages/standardjs/test/standardjs_test.js

```diff
@@ -9,37 +9,25 @@ test('loads preset', t => {
 });
 
 test('uses preset', t => {
-  t.notThrows(() => Neutrino().use(mw()));
+  t.notThrows(() => new Neutrino().use(mw()));
 });
 
 test('uses with options', t => {
-  t.notThrows(() => Neutrino().use(mw(), options));
+  t.notThrows(() => new Neutrino().use(mw(), options));
 });
 
 test('instantiates', t => {
-  const api = Neutrino();
+  const api = new Neutrino();
 
   api.use(mw());
 
   t.notThrows(() => api.config.toConfig());
 });
 
 test('instantiates with options', t => {
-  const api = Neutrino();
+  const api = new Neutrino();
 
   api.use(mw(), options);
 
   t.notThrows(() => api.config.toConfig());
 });
-
-test('exposes lint command', t => {
-  const api = Neutrino();
-
-  api.use(mw());
-
-  t.is(typeof api.commands.lint, 'function');
-});
-
-test('exposes eslintrc config', t => {
-  t.is(typeof Neutrino().use(mw()).call('eslintrc'), 'object');
```

Comment:
Should we convert this and the other "expose XYZ" to checking that `...output('...')` works?